config.js

```diff
@@ -1,5 +1,13 @@
-// Rename this file to env.js 
-// and update the API_ID for a valid one
-export default {
-    API_ID: 'BOT_API'
-}
\ No newline at end of file
+require('dotenv').config();
+
+module.exports = {
+  firebase: {
+    apiKey: process.env.FIREBASE_API_KEY,
+    authDomain: 'pingchannelbot.firebaseapp.com',
+    databaseURL: 'https://pingchannelbot.firebaseio.com',
+    storageBucket: 'pingchannelbot.appspot.com',
+  },
+  apiId: process.env.API_ID,
+  currentPort: process.env.PORT,
```

Comment:
I would move ``|| 3000`  then we will only have this responsibility in one place.